src/ur/ur_driver.cpp

```diff
@@ -144,13 +144,23 @@ void UrDriver::init(const UrDriverConfiguration& config)
 
   const control::PDControllerGains pd_gains = control::getPdGainsFromRobotType(robot_type);
   std::stringstream pd_gains_ss;
-  pd_gains_ss << "struct(kp=" << pd_gains.kp << ", kd=" << pd_gains.kd << ")";
+  if (robot_version_ < urcl::VersionInformation::fromString("5.10.0"))
```

Comment:
Are we still support software version below 5.10? Would be better to check whether it is an cb3 robot?

Reply:
It seems reasonable I also just realized that we support from 5.9.3 an onwards so it is a valid check.